### ISSUE TITLE
Header Template

### ISSUE THREAD

**Issue by vragosta** [vragosta/dracobit.io#188]

**Comment by vragosta** [comment]
@mannibale do you think the bottom image ( below 530px ), should have its active state span 100% width?

Meaning, the browse button is highlighted with a light blue background. Should that blue background span 100% width?

**Comment by mannibale** [comment]
@vragosta Yes, I believe it should span 100% within the hamburger dropdown. Having it float there w/o it touching the bounds of the outer box makes it feel too secular.

**Comment by vragosta** [comment]
@mannibale thank you. What about the centering of the menu items? Should they be left justified?

**Comment by mannibale** [comment]
@vragosta No, they should be fine remaining where they are. Centered looks good.
